examples/seldon_custom_pytorch_deployment/README.md

```diff
@@ -0,0 +1,531 @@
+
+zenml model-deployer register seldon_gcp --type=seldon \
+  --kubernetes_context=gke_cluster-1 --kubernetes_namespace=kubeflow \
+  --base_url=http://34.139.128.148
+zenml artifact-store register gcp --type gcp --path gs://mybucket
+zenml container-registry register gcp --type=default --uri=gcr.io/container-registry
+zenml metadata-store register gcp --type=kubeflow
+zenml orchestrator register gcp --type=kubeflow --kubernetes_context=gke_cluster-1 --synchronous=True
+zenml stack register gcp_kubeflow_seldon -m gcp -a gcp -o gcp -d seldon_gcp -c gcp
+zenml stack set gcp_kubeflow_seldon
+
+```
+
+### Run the project
+To run the continuous deployment pipeline:
+
+```shell
+python run.py --secret seldon-rclone-secret --deploy
+```
+
+Example output when run with the local orchestrator stack:
+
+```
+zenml/seldon_deployment$ python run.py --secret seldon-rclone-secret --deploy --min-accuracy 0.80
+
+2022-04-06 15:40:28.903233: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
+2022-04-06 15:40:28.903253: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
+Creating run for pipeline: `continuous_deployment_pipeline`
+Cache disabled for pipeline `continuous_deployment_pipeline`
+Using stack `local_with_aws_storage` to run pipeline `continuous_deployment_pipeline`...
+Step `importer_mnist` has started.
+INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
+Step `importer_mnist` has finished in 20.012s.
+Step `normalizer` has started.
+Step `normalizer` has finished in 24.177s.
+Step `sklearn_trainer` has started.
+Step `sklearn_trainer` has finished in 23.150s.
+Step `sklearn_evaluator` has started.
+Step `sklearn_evaluator` has finished in 10.511s.
+Step `deployment_trigger` has started.
+Step `deployment_trigger` has finished in 4.965s.
+Step `seldon_model_deployer` has started.
+INFO:asyncio:Loading last service deployed by step model_deployer and pipeline continuous_deployment_pipeline...
+Creating a new Seldon deployment service
+Seldon deployment service started and reachable at:
+    http://abb84c444c7804aa98fc8c097896479d-377673393.us-east-1.elb.amazonaws.com/seldon/zenml-workloads/zenml-1
+6241824-7e17-42d8-bed3-070b51ba29d2/api/v0.1/predictions
+
+Step `seldon_model_deployer` has finished in 39.095s.
+Pipeline run `continuous_deployment_pipeline-06_Apr_22-15_40_31_886832` has finished in 2m2s.
+The Seldon prediction server is running remotely as a Kubernetes service and accepts inference requests at:
+    http://abb84c444c7804aa98fc8c097896479d-377673393.us-east-1.elb.amazonaws.com/seldon/zenml-workloads/zenml-1
+6241824-7e17-42d8-bed3-070b51ba29d2/api/v0.1/predictions
+To stop the service, re-run the same command and supply the `--stop-service` argument.
```

Comment:
This is quite long, could you remove unnecessary outputs and just highlight the important bits

Reply:
Yes, i agree. it's done